addons/web_editor/views/ir_attachment_views.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+     <record id="view_attachment_form_inherit_web_editor" model="ir.ui.view">
+        <field name="name">ir.attachment.form.inherit.web_editor</field>
+        <field name="model">ir.attachment</field>
+        <field name="inherit_id" ref="base.view_attachment_form"/>
+        <field name="arch" type="xml">
+            <field name="name" position="before">
+                <field class="me-2" name="is_favorite" widget="boolean_favorite" nolabel="1"/>
```

Comment:
It's IMHO lacking the concept of "editor" (media gallery) favorite, otherwise you don't really know what favorites is used for when going to the list/form view of attachment.
We will discuss all that once we got the approval of framework-py.
Maybe it's not even worth adding it here but only use the field in the media gallery in the builder and do nothing in the backend?